autogen/agentchat/conversable_agent.py

```diff
@@ -304,7 +308,7 @@ def send(
         recipient: Agent,
         request_reply: Optional[bool] = None,
         silent: Optional[bool] = False,
-    ) -> bool:
+    ):
```

Comment:
How about "-> None" rather than empty?

@sonichi  Which way is preferred? Ignore the return type or declare it as "None".

Reply:
I think the convention is to have no return type in this case.